spec/lib/auth0/api/authentication_endpoints_spec.rb

```diff
@@ -330,6 +330,14 @@
     end
   end
 
+  context '.userinfo' do
+    it { expect(@instance).to respond_to(:userinfo) }
```

Comment:
If you setup something like `subject { @instance }`, you could even shorten this to `it { is_expected.to respond_to(:user_info) }`, I believe. Not suggesting you need to do this, just showing you some fun RSpec stuff 👍